Code/Framework/AzNetworking/AzNetworking/Framework/INetworking.h

```diff
@@ -46,6 +46,16 @@ namespace AzNetworking
         //! @return pointer to the instantiated network interface, or nullptr on error
         virtual INetworkInterface* CreateNetworkInterface(const AZ::Name& name, ProtocolType protocolType, TrustZone trustZone, IConnectionListener& listener) = 0;
 
+        //! Creates a new network interface instance with the provided parameters that operates on the SystemTickBus.
+        //! System Network Interfaces are suitable for cases where updates occur regardless of application state such as debug utilities.
```

Comment:
I like this PR comment about _"This allows the creation of Network Interfaces that will run even if the application in question is in the background."_ Wouldn't mind seeing that added after this comment line.